Sources/Views/Products/Detail/ProductAttributes/AttributeTableViewCell.swift

```diff
@@ -0,0 +1,125 @@
+        let infoImageView = stackView.arrangedSubviews.last
+        stackView.removeAllViews()
+
+        let attributeView = AttributeView.loadFromNib()
+
+        attributeView.configure(attribute)
+
+        configureGestureRecognizer()
+        stackView.addArrangedSubview(attributeView)
+        if let iiv = infoImageView {
+            stackView.addArrangedSubview(iiv)
+        }
+
+        delegate = attributeTableRow.delegate
+    }
+
+    override func dismiss() {
+        super.dismiss()
+        stackView.arrangedSubviews.forEach {
+            if let view = $0 as? IngredientsAnalysisView {
+                view.removeGestureRecognizer()
+            }
+        }
+        stackView.removeAllViews()
+    }
+
+    func configureGestureRecognizer() {
+        self.gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(didTap))
+        if let gestureRecognizer = self.gestureRecognizer {
+            self.addGestureRecognizer(gestureRecognizer)
+            self.isUserInteractionEnabled = true
+        }
+    }
+
+    func removeGestureRecognizer() {
+        if let validGesture = self.gestureRecognizer {
+            self.removeGestureRecognizer(validGesture)
+            self.gestureRecognizer = nil
+        }
+    }
+
+    @objc func didTap(_ sender: UITapGestureRecognizer) {
+        print("Attribute tapped")
+        guard let attribute = attribute else {
+            return
+        }
+
+        let attributeView = AttributeView.loadFromNib()
+        attributeView.configure(attribute)
+
+        delegate?.attributeTableViewCellTapped(self, attributeView)
+    }
+}
+
+class AttributeBLTNPageItem: BLTNPageItem {
+    var attribute: Attribute?
+    var iconImageBackgroundColor: UIColor?
+    var attributeView: AttributeView?
+
+    override func makeHeaderViews(with interfaceBuilder: BLTNInterfaceBuilder) -> [UIView]? {
+        let containerA = UIView()
+        containerA.backgroundColor = iconImageBackgroundColor
+
+        if let attributeV = attributeView {
+            containerA.addSubview(attributeV)
+
+            constrain(attributeV, containerA) { (attributeV, containerA) in
+                containerA.width == attributeV.width
+                containerA.height == attributeV.height
+                attributeV.edges == containerA.edges
+
+            }
+
+            var views: [UIView] = [containerA]
+
+            if let attribute = attribute {
+                let descriptionLabel = UILabel()
+                descriptionLabel.numberOfLines = 0
+                descriptionLabel.textAlignment = .center
+                descriptionLabel.font = UIFont.boldSystemFont(ofSize: 17)
+
+                descriptionLabel.text = attribute.descriptionLong ?? attribute.descriptionShort ?? "empty description"
+                views.append(descriptionLabel)
+            }
+            return views
+        }
+        return nil
+    }
```

Comment:
I'm sorry @philippeauriach, I thought I removed this class and all my BLTNboard stuff. Now I see why you thought I was mixing them!